boards/arm/phy62xx/phy6222/src/bringup.c

```diff
@@ -125,16 +137,26 @@ int phy62xx_bringup(void)
 
       /* Mount the LittleFS file system */
 
-      ret = nx_mount("/dev/mtd", "/mnt/lfs", "littlefs", 0,
+      ret = nx_mount("/dev/mtd", "/data", "littlefs", 0,
                 "forceformat");
```

Comment:
the "forceformat" should be modified to "autoformat"

Reply:
ok , i will fix it.